### DISCUSSION TITLE
Error on deployed project

### DISCUSSION THREAD

**Discussion by oPaozinh0** [SamPandey001/Zero-Two-MD#10] [Q&A] [answered]
Hi! I'm trying to deploy and run the project, but I have this error on heroku logs:

`

> 2022-06-27T15:20:01.954928+00:00 app[web.1]: 0|index  | Connected to the MongoDb Darling 
> 
> 2022-06-27T15:20:02.051155+00:00 app[web.1]: 0|index  | Error: Cannot find module 'pastebin-js'
> 
> 2022-06-27T15:20:02.051173+00:00 app[web.1]: 0|index  | Require stack:
> 
> 2022-06-27T15:20:02.051191+00:00 app[web.1]: 0|index  | - /app/lib/colab.js
> 
> 2022-06-27T15:20:02.051208+00:00 app[web.1]: 0|index  | - /app/index.js
> 
> 2022-06-27T15:20:02.051226+00:00 app[web.1]: 0|index  |     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
> 2022-06-27T15:20:02.051244+00:00 app[web.1]: 0|index  |     at Module.Hook._require.Module.require (/app/.heroku/node/lib/node_modules/pm2/node_modules/require-in-the-middle/index.js:61:29)
> 
> 2022-06-27T15:20:02.051261+00:00 app[web.1]: 0|index  |     at require (node:internal/modules/cjs/helpers:102:18)
> 
> 2022-06-27T15:20:02.051277+00:00 app[web.1]: 0|index  |     at Object.<anonymous> (/app/lib/colab.js:4:518)
> 
> 2022-06-27T15:20:02.051295+00:00 app[web.1]: 0|index  |     at Module._compile (node:internal/modules/cjs/loader:1105:14)
> 
> 2022-06-27T15:20:02.051312+00:00 app[web.1]: 0|index  |     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
> 
> 2022-06-27T15:20:02.051331+00:00 app[web.1]: 0|index  |     at Module.load (node:internal/modules/cjs/loader:981:32)
> 
> 2022-06-27T15:20:02.051347+00:00 app[web.1]: 0|index  |     at Function.Module._load (node:internal/modules/cjs/loader:822:12)
> 
> 2022-06-27T15:20:02.051364+00:00 app[web.1]: 0|index  |     at Module.require (node:internal/modules/cjs/loader:1005:19)
> 
> 2022-06-27T15:20:02.051381+00:00 app[web.1]: 0|index  |     at Module.Hook._require.Module.require (/app/.heroku/node/lib/node_modules/pm2/node_modules/require-in-the-middle/index.js:80:39)
> 
> 2022-06-27T15:20:02.051399+00:00 app[web.1]: 0|index  |     at require (node:internal/modules/cjs/helpers:102:18)
> 
> 2022-06-27T15:20:02.051417+00:00 app[web.1]: 0|index  |     at Object.<anonymous> (/app/index.js:85:1718)
> 
> 2022-06-27T15:20:02.051429+00:00 app[web.1]: 0|index  |     at Module._compile (node:internal/modules/cjs/loader:1105:14)
> 
> 2022-06-27T15:20:02.051446+00:00 app[web.1]: 0|index  |     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
> 
> 2022-06-27T15:20:02.051464+00:00 app[web.1]: 0|index  |     at Module.load (node:internal/modules/cjs/loader:981:32)
> 
> 2022-06-27T15:20:02.051483+00:00 app[web.1]: 0|index  |     at Function.Module._load (node:internal/modules/cjs/loader:822:12) {
> 
> 2022-06-27T15:20:02.051500+00:00 app[web.1]: 0|index  |   code: 'MODULE_NOT_FOUND',
> 
> 2022-06-27T15:20:02.051518+00:00 app[web.1]: 0|index  |   requireStack: [ '/app/lib/colab.js', '/app/index.js' ]
> 
> 2022-06-27T15:20:02.051530+00:00 app[web.1]: 0|index  | }
> 
> 2022-06-27T15:20:02.061005+00:00 app[web.1]: PM2      | App [index:0] exited with code [1] via signal [SIGINT]
> 
> 2022-06-27T15:20:02.063556+00:00 app[web.1]: PM2      | App [index:0] starting in -fork mode-
> 
> 2022-06-27T15:20:02.068437+00:00 app[web.1]: PM2      | App [index:0] online

`

**Comment by SamPandey001** [comment, marked as answer]
Hii @zBreadz ,Thanks for deployng Zero-Two.
The error has been fixed,just fetch upstream and deploy
Thanks for pinging us errors.
